src/cgls/cgls.c

```diff
@@ -50,6 +51,7 @@ static void help(void) {
                "     --version        Show package version\n"
                "     --no-pager       Do not pipe output into a pager\n"
                "  -a --all            Show all groups, including empty\n"
+               "  -u --unit           Show the subtrees of specifified units\n"
```

Comment:
hmm, so the other tools we have that that have a --unit option usually expect a parameter to it, i.e. "journalctl --unit=foobar"... 

I think to make this less surprising we should probably do this similar here. Maybe an OK idea would be to use the "optional" getopt type, so that `--unit=foo` works, but also `--unit foo bar`, if you follow what I mean?